.github/workflows/agw-docker-load-test.yml

```diff
@@ -12,14 +12,10 @@
 name: AGW docker load test
 
 on:
-  workflow_run:
-    workflows:
-      - build-all
+  push:
```

Comment:
- For this workflow the trigger on workflow completion is required in the current state. It seems to rely on the commit hash in [(new) line 27](https://github.com/magma/magma/pull/14227/files#diff-32f79c1ad74bcf681f59b6b22e69ea64ccdb521a11b45d3f468502ae35ec6a59R27).
I did not check, if there are dependencies deep down in the Ansible scripts. However, at least that occurance would need to be somehow adopted.
- More general, we currently don't build AMIs and afaik, there was an effort to remove the Magma AMIs from AWS Marketplace. So I am not sure, if this workflow is valid any more at all.
I would just leave it out of scope of the current change and _optionally_ create a ticket about what to do with it.
- Just looked again. This workflow has never run successfully. Did not investigate. Better not touch now and create a ticket for cleanup.

Reply:
Will exclude this workflow and create an issue to clean up.